apps/modernization-ui/src/apps/page-builder/page/management/edit/tabs/ManageTabs/ManageTabs.tsx

```diff
@@ -95,24 +115,53 @@ export const ManageTabs = ({ pageId, tabs, onAddSuccess }: Props) => {
                 }
                 modalBody={
                     <div className={styles.modalBody}>
-                        {message && (
+                        {message ? (
                             <AlertBanner type={message.type} expiration={message.expiration}>
                                 {message.message}
                             </AlertBanner>
-                        )}
-                        {addEdit ? <AddEditTab tabData={null} onChanged={setNewTab} /> : null}
+                        ) : null}
+                        <AlertBanner type={'info'} expiration={5000}>
+                            <p>
+                                Tabs from a template cannot be reordered, edited or deleted. Only new tabs that are
+                                added using the button above can be managed.
+                            </p>
```

Comment:
Issue: verify with Jared - this message is not true.

Reply:
Okay, designs have been updated.